src/haywire/http_connection.h

```diff
@@ -1,18 +1,2 @@
 #pragma once
-#include "uv.h"
-#include "http_parser.h"
-#include "http_request.h"
-
-typedef struct
-{
-    uv_tcp_t stream;
-    http_parser parser;
-    uv_write_t write_req;
-    http_request* request;
-    char current_header_key[1024];
-    int current_header_key_length;
-    char current_header_value[1024];
-    int current_header_value_length;
-    int keep_alive;
-    int last_was_value;
-} http_connection;
+#include "http.h"
```

Comment:
@kellabyte seems like `http_connection.h` is just a proxy for `http.h`, no? If so, why not just include `http.h` directly?